libcaf_test/caf/test/scenario.test.cpp

```diff
@@ -25,6 +25,23 @@ SCENARIO("a scenario may not contain a section") {
   });
   check(!entered_section);
 }
+
+// Regression test for #1799: Scenarios should not parse parameter placeholders.
+SCENARIO("scenario with <foo> syntax in description") {
+  GIVEN("given a <description>") {
+    WHEN("when <parsing> the raw description") {
+      THEN("then we copy the <description> verbatim") {
+        auto i = ctx_->call_stack.begin();
+        check_eq((*i++)->description(),
+                 "scenario with <foo> syntax in description");
+        check_eq((*i++)->description(), "given a <description>");
+        check_eq((*i++)->description(), "when <parsing> the raw description");
+        check_eq((*i++)->description(),
+                 "then we copy the <description> verbatim");
```

Comment:
Unsafe, and IMHO a bit hard to follow due to the iterator indirections. I'd prefer something like this:

```suggestion
        auto cs = ctx_->call_stack;
        require_eq(cs.size(), 4u);
        check_eq(cs[0]->description(),
                 "scenario with <foo> syntax in description");
        check_eq(cs[1]->description(), "given a <description>");
        check_eq(cs[2]->description(), "when <parsing> the raw description");
        check_eq(cs[3]->description(),
                 "then we copy the <description> verbatim");
```